autoreject/utils.py

```diff
@@ -229,6 +231,61 @@ def interpolate_bads(inst, picks, reset_bads=True, mode='accurate'):
     return inst
 
 
+def _interpolate_bads_eeg(inst, picks, verbose=None):
```

Comment:
I would add a test to check if it gives the same result as MNE ... at least when `picks` is None